### PR TITLE
design layout fix home

### PR DESCRIPTION
**Pull request by namkidd**

### CODE REVIEW

**Comment by namkidd** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -605,7 +607,28 @@ table {
...
+
+.container-fluid {
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.navbar-inverse {
+  background-color: #222;
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/custom.scss]: dùng `variable in Sass`

**Comment by huongnguyenmta** [comment]
commented

**Comment by namkidd** [comment]
ready

**Pull request merged**